fin_cander/fin_cander/settings.py

```diff
@@ -93,10 +94,18 @@
         "PASSWORD": os.environ.get("POSTGRES_PASSWORD"),
         "HOST": os.environ.get("POSTGRES_HOST"),
         "PORT": os.environ.get("POSTGRES_PORT"),
-    }  
+    },
+    "nonrel": {
+        "ENGINE": "djongo",
+        "NAME": os.environ.get('MONGO_DB_NAME'),
+        "CLIENT": {
+            'host': 'mongodb://root:root@mongo/admin?authSource=admin'
```

Comment:
you should use envs here as well

Reply:
@kgarbacinski done 

`         'host': f"mongodb://{os.environ.get('MONGO_DB_USERNAME')}:{os.environ.get('MONGO_DB_PASSWORD')}@"
                    f"{os.environ.get('MONGO_DB_HOST')}/{os.getenv('MONGO_DB_NAME')}?authSource=admin"`